launcher/Application.cpp

```diff
@@ -1,7 +1,10 @@
 // SPDX-License-Identifier: GPL-3.0-only
 /*
- *  PolyMC - Minecraft Launcher
+ *  Prism Launcher
  *  Copyright (C) 2022 Sefa Eyeoglu <contact@scrumplex.net>
+ *  Copyright (C) 2022 Tayou <tayou@gmx.net>
+ *
+ *  PolyMC - Minecraft Launcher
  *  Copyright (C) 2022 Lenny McLennington <lenny@sneed.church>
```

Comment:
Just change `PolyMC` to `Prism Launcher` and keep the existing list of copyright holders

```suggestion
 *  Prism Launcher - Minecraft Launcher
 *  Copyright (C) 2022 Sefa Eyeoglu <contact@scrumplex.net>
 *  Copyright (C) 2022 Lenny McLennington <lenny@sneed.church>
 *  Copyright (C) 2022 Tayou <tayou@gmx.net>
```

Reply:
Accidental approval lol

Reply:
already changed locally. 
the QList stuff we talked about last afternoon broke some things, which is why i didn't commit & push yet.
I think I'm just gonna not do that for now, its staying a std::vector ^^